samples/subsys/usb/console/Kconfig

```diff
@@ -0,0 +1,4 @@
+config USB_DEVICE_PID
+	default USB_PID_CONSOLE_SAMPLE
```

Comment:
I'd still go for putting the values directly in these files, but I'll leave it up to others.